src/MICore/CommandFactories/gdb.cs

```diff
@@ -33,6 +33,11 @@ public override bool SupportsStopOnDynamicLibLoad()
             return true;
         }
 
+        public override bool SupportsChildProcessDebugging()
+        {
+            return true;
```

Comment:
gdb (under Cygwin anyway) does not catch the fork. Unfortunately it also doesn't report an error of any kind when you try (except in an output string after run is executed). I added this check
_launchOptions is LocalLaunchOptions && RuntimeInformation.IsOSPlatform(OSPlatform.Windows)